### ISSUE TITLE
Release Cycle

### ISSUE THREAD

**Issue by arcfide** [OpenCDE/opencde#16]
We have not yet come to a conclusion about how we want to do a release cycle. I am a fan of fixed release cycles and consistent, if at times slow development. I like the OpenBSD model of development, but with DVCS we can do better.

I have created [Release Cycle](../wiki/Release-Cycle) for documenting our process.

**Comment by woomia** [comment]
Yes, we talked about this on the OpenCDE forums but I'll add my two cents
right now,

On Sun, Jan 22, 2012 at 9:01 PM, arcfide <
reply@reply.github.com

**Comment by arcfide** [comment]
We can just link to the discussion: http://devio.us/~kpedersen/forums/viewtopic.php?id=1231
